File: Array/Compare 2 Array/Compare_2_array.java
import java.util.*;
class Compare_2_array{
    public static void main(String[] args){
        Scanner sc=new Scanner(System.in);
        int a[]=new int[5];
        int b[]=new int[5];
        int i,f=0;
        System.out.print("Enter 5 number for 1st array : ");
        for (i=0;i<5;i++)
            a[i]=sc.nextInt();
        System.out.print("Enter 5 number for 2nd array : ");
        for (i=0;i<5;i++)
            b[i]=sc.nextInt();
        System.out.print("1st Array : ");
        for (i=0;i<5;i++)
            System.out.print(a[i]+" ");
        System.out.print("\n2nd Array : ");
        for (i=0;i<5;i++)
            System.out.print(b[i]+" ");
        for(i=0;i<5;i++)
            if(a[i]!=b[i]){
                f=1;
                break;
            }
        if(f==0)
            System.out.print("\nBoth the array are same.");
        else
            System.out.print("\nBoth the array are different.");       
    }
}

File: Array/Compare 2 Array/Compare_2_array.c
#include<stdio.h>
void main(){
    int a[5],b[5];
    int i,f=0;
    printf("Enter 5 number for 1st array : ");
    for (i=0;i<5;i++)
        scanf("%d",&a[i]);
    printf("Enter 5 number for 2nd array : ");
    for (i=0;i<5;i++)
        scanf("%d",&b[i]);
    printf("1st Array : ");
    for (i=0;i<5;i++)
        printf("%d ",a[i]);
    printf("\n2nd Array : ");
    for (i=0;i<5;i++)
        printf("%d ",b[i]);
    for(i=0;i<5;i++)
        if(a[i]!=b[i]){
            f=1;
            break;
        }
    if(f==0)
        printf("\nBoth the array are same.");
    else
        printf("\nBoth the array are different.");   
}